code/datums/martial/sleeping_carp.dm

```diff
@@ -13,13 +13,13 @@
 	. = ..()
 	if(!.)
 		return
-	target.add_traits(list(TRAIT_NOGUNS, TRAIT_HARDLY_WOUNDED, TRAIT_NODISMEMBER), SLEEPING_CARP_TRAIT)
+	target.add_traits(list(TRAIT_NOGUNS, TRAIT_HARDLY_WOUNDED, TRAIT_NODISMEMBER, TRAIT_HEAVY_SLEEPER), SLEEPING_CARP_TRAIT)
```

Comment:
Should make this list of traits a list on the datum itself, that we pass to `add` and `remove`. Makes it easier to modify going forward

Reply:
sure